lib/active_record/enum_override.rb

```diff
@@ -20,8 +20,15 @@ def sql_enum(name, options={})
       detect_enum_conflict!(name, "#{name}=")
 
       attr = attribute_alias?(name) ? attribute_alias(name) : name
-      decorate_attribute_type(attr, :enum) do |subtype|
-        EnumType.new(attr, enum_values, subtype)
+
+      if Rails.version >= '6.1'
+        decorate_attribute_type(name) do |subtype|
```

Comment:
fwiw, a very similar fix on the `enumerize` gem recommends using `name.to_s` instead of just `name`.
https://github.com/brainspec/enumerize/pull/365/files